### DISCUSSION TITLE
No translate when move all to folder src

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vinissimus/next-translate#867] [Q&A]
Hello, I am trying use next-translate, everything work fine but when I move all file to src folder, next-transalte won't work anymore. Can you fix this?

**Comment by aralroca** [comment]
Look at this example https://github.com/vinissimus/next-translate/tree/master/examples/complex is using `src`. Probably you need to overwrite the `loadLocaleFrom` from `i18n.js`
